src/pages/resultsView/download/CaseAlterationTable.tsx

```diff
@@ -141,14 +144,18 @@ export function computeAlterationTypes(alterationData: ICaseAlteration[]): strin
 export function getDisplayStyle(value: string): object {
     switch(value) {
         case "no alteration":
-            return {whiteSpace: "nowrap" , color: "orange"};
+            return {whiteSpace: "nowrap" , color: "gray"};
```

Comment:
this should really be in css and applied via classname.   This is especially important for elements which are repeated many times in DOM as their is a performance cost.  Just as a general rule.